DscResources/MSFT_xMySqlDatabase/MSFT_xMySqlDatabase.schema.mof

```diff
@@ -1,9 +1,10 @@
 [ClassVersion("1.0.0"),FriendlyName("xMySqlDatabase")] 
 class MSFT_xMySqlDatabase : OMI_BaseResource
 {
-    [Key, Description("Name of the database.")] String Name;
-    [Required, Description("Should the database be present or absent."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
-    [Required, Description("The root credential that is used to install MySql server."), EmbeddedInstance("MSFT_Credential")] String ConnectionCredential;
+    [Key, Description("Name of the database.")] String DatabaseName;
```

Comment:
Resource name is xMySqlDatabase , so the property "Name" would mean the database name ( this resource does not use or refer anyhow to any other "name" ) , the property 's  description also says so "Name of the database", why would renaming Name to be DatabaseName be needed? It can also break existing dsc configurations that depend on this resource. I would suggest to leave the property name as "Name".